homeassistant/components/google_weather/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: done
+  reauthentication-flow: todo
+  test-coverage: done
+
+  # Gold
+  devices: done
+  diagnostics: todo
+  discovery-update-info:
+    status: exempt
+    comment: No discovery.
+  discovery:
+    status: exempt
+    comment: No discovery.
+  docs-data-update: done
+  docs-examples: done
+  docs-known-limitations: done
+  docs-supported-devices: done
+  docs-supported-functions: done
+  docs-troubleshooting: done
+  docs-use-cases: done
+  dynamic-devices:
+    status: exempt
+    comment: N/A
+  entity-category: done
+  entity-device-class: done
+  entity-disabled-by-default: done
+  entity-translations: done
+  exception-translations: todo
+  icon-translations: done
+  reconfiguration-flow: todo
+  repair-issues:
+    status: exempt
+    comment: No repairs.
+  stale-devices:
+    status: exempt
+    comment: N/A
+
+  # Platinum
+  async-dependency: done
+  inject-websession: done
+  strict-typing: done
```

Comment:
```suggestion
  strict-typing: todo
```

It doesn't have strict typing in the coordinator data.

Reply:
Which lines? I've enabled strict typing in mypy.ini and mypy doesn't complain. Isn't that enough? Are you referring to the `Any` in the `dict[str, Any]`? I just searched for other integrations that have `strict-typing: done` and a similar dict with Any and found a few, e.g. [vodafone_station](https://github.com/home-assistant/core/blob/a102eaf0cddfd72d654606d3497bb408ab599984/homeassistant/components/vodafone_station/coordinator.py#L42), [fronius](https://github.com/home-assistant/core/blob/a102eaf0cddfd72d654606d3497bb408ab599984/homeassistant/components/fronius/coordinator.py#L38), [enphase_envoy](https://github.com/home-assistant/core/blob/a102eaf0cddfd72d654606d3497bb408ab599984/homeassistant/components/enphase_envoy/coordinator.py#L38)

Reply:
I prefer to enforce it more by using typed models, but it seems like it's not a requirement for the rule to be set to done. I'm going to mark this as resolved, but know that you can improve by adding typed models. :)

Reply:
(If this is blocking approval and merge, I'll happily make it todo and continue the discussion separately.)